<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amazon Spider</string>
  </property>
  <widget class="QLabel" name="testName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;amazon spider 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入需要爬的网址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入此时查询的关键字</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="UrlText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="KeywordText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="takeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cleanButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cleanButton</sender>
   <signal>clicked()</signal>
   <receiver>KeywordText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleanButton</sender>
   <signal>clicked()</signal>
   <receiver>UrlText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>takeButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>submitRequest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UrlText</sender>
   <signal>textChanged()</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>KeywordText</sender>
   <signal>textChanged()</signal>
   <receiver>Dialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submitRequest()</slot>
 </slots>
</ui>
